libs/sarif/src/schema/properties.rs

```diff
@@ -12,19 +12,22 @@ use serde::{Deserialize, Serialize};
     derive(Serialize, Deserialize),
     serde(rename_all = "camelCase")
 )]
-pub struct PropertyBag {
+pub struct PropertyBag<'s> {
     /// A set of distinct strings that provide additional information.
-    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "BTreeSet::is_empty"))]
-    pub tags: BTreeSet<Cow<'static, str>>,
+    #[cfg_attr(
+        feature = "serde",
+        serde(skip_serializing_if = "BTreeSet::is_empty", borrow)
+    )]
+    pub tags: BTreeSet<Cow<'s, str>>,
 
     /// A dictionary, each of whose keys specifies a distinct additional information element and
     /// each of whose values provides the information.
     #[cfg(feature = "serde")]
-    #[serde(flatten)]
-    pub additional: BTreeMap<Cow<'static, str>, serde_json::Value>,
+    #[serde(flatten, borrow)]
```

Comment:
Any specific reason why additional properties are gated under the `serde`?

Reply:
It requires serde-json’s Value. I’m about to change this in a future PR